src/transformState.js

```diff
@@ -63,6 +63,27 @@
  */
 function transformState(state, transforms) {
   // write code here
+  const resObj = state;
+
+  for (const obj of transforms) {
```

Comment:
obj дуже абстрактна назва, якщо ти перебираєш масив transforms то в однині це transform
```suggestion
  for (const transform of transforms) {
```

а можна одразу деструкткризацію робити

```suggestion
  for (const { operation, properties } of transforms) {
```

Reply:
Добре, цього ще на знав, дякую.